sherpa/astro/ui/tests/test_astro_ui_fluxes.py

```diff
@@ -2101,8 +2100,8 @@ def test_sample_flux_errors(make_data_path, clean_astro_ui,
     elo = fmed - flsig
     ehi = fusig - fmed
     assert fmed == pytest.approx(77.27966775437665)
-    assert elo == pytest.approx(9.86485842683382)
-    assert ehi == pytest.approx(11.40548578502279)
+    assert elo == pytest.approx(9.773389936230018)
```

Comment:
All I did was grab the expected value and copy it over to the test. While I can understand an argument for saying "only bother with giving values we use in the comparison", I can also come up with 

a) I don't have time to sit down for each value and validate the precision that can be used
b) if we ever decide to switch to a different approximation routine (or pytest.approx changes) then it would be annoying to have to re-calculate all these values
c) having the full value let's us see if there are subtle differences when reviewing code changes (this is definitely clutching at straws)

I would argue it's primarily issue a.